cilium/cilium
# Fix http gsg app yaml and delete old app yaml files

**Summary of changes**:
- fixed the api version spec for Deployment object in the app yaml. This is for backward compatibility with K8s version before 1.9
- removed the old demo app files (demo.yaml and policy files)

Fixes: [3928](https://github.com/cilium/cilium/issues/3928)

**How to test (optional)**:
create minikube with version 1.8 
create the app using the yaml. all pods and svcs should get created without errors

create minikube with version 1.9
create the app using the yaml. all pods and svcs should get created without errors

Signed-off-by: Arvind Soni <arvindsoni@gmail.com>
